plugins/lookup/aws_secret.py

```diff
@@ -25,6 +25,10 @@
   _terms:
     description: Name of the secret to look up in AWS Secrets Manager.
     required: True
+  bypath:
+    description: A boolean to indicate whether the parameter is provided as a hierarchy.
+    default: false
+    type: boolean
```

Comment:
```suggestion
    type: bool
    version_added: '1.4.0'
```

Reply:
@felixfontein should this have been caught by the sanity tests?

Reply:
1. There are no plugin sanity tests yet (ansible/ansible#71734 hasn't been merged).
2. According to ansible/ansible#71796 the long-term goal is to support both the short and long form for plugins and modules. So both `bool` and `boolean` would be correct here.

Reply:
I switched the other `boolean` to `bool` as well.